core/workspace_comment/workspace_comment.js

```diff
@@ -163,3 +179,60 @@ Blockly.WorkspaceComment.prototype.setContent = function(content) {
     this.text_ = content;
   }
 };
+
+/**
+ * Encode a comment subtree as XML with XY coordinates.
+ * @param {boolean} opt_noId True if the encoder should skip the comment id.
```

Comment:
Is it worth commenting this function is identical to the one for Blocks in xml.js?